controllers/managedfusion_controller.go

```diff
@@ -669,8 +687,8 @@ func (r *ManagedFusionReconciler) reconcileMonitoringResources() error {
 	r.Log.Info("reconciling monitoring resources")
 
 	podMonitorList := promv1.PodMonitorList{}
-	if err := r.list(&podMonitorList); err != nil {
-		return fmt.Errorf("Could not list pod monitors: %v", err)
+	if err := r.list(&podMonitorList, client.InNamespace(r.Namespace)); err != nil {
```

Comment:
Why should we list podMonitors, serviceMonitors and promRules in single namespace only(the namespace where monitoring stack is deployed)?
We should add labels to the monitoring resources that are created by the offering as well.
Example: cephPromRules are created by ocs-operator so our prometheus should be able to read those rules and raise alerts, for this to happen, we should add label to the cephPromRules.

Reply:
Thinking about it again, it does not make sense to have this here at all. Each offering might have its own way of producing/creating the monitoring resources and might even reconcile back any changes to them.
For DF, OCS operator is adding he ability to ask it to add a desired label to the monitoring resources. This means the code above is not needed after we start deploying with ocs-operator 4.12. In the meantime, I think this code should move into `datafoundation.go` code, but that is for another PR.

Reply:
I have reverted the changes to be moved to `datafoundation.go` at a future Pr